source/val/function.h

```diff
@@ -174,6 +174,9 @@ class Function {
   /// Returns the block predecessors function for the augmented CFG.
   GetBlocksFunction AugmentedCFGPredecessorsFunction() const;
 
+  /// Returns the control flow nesting depth of the given basic block.
+  int GetBlockDepth(BasicBlock* bb);
```

Comment:
I see there's a reasonable set of definitions there. That will do for those who are curious.